aot/src/main/java/com/dylibso/chicory/experimental/aot/AotCompiler.java

```diff
@@ -84,6 +86,8 @@ public final class AotCompiler {
     private static final MethodType MACHINE_CALL_METHOD_TYPE =
             methodType(long[].class, Instance.class, Memory.class, int.class, long[].class);
 
+    private static final int MAX_MACHINE_CALL_METHODS = 1024; // must be power of two
```

Comment:
Is the current limit this low?
This is the reference limit if I understand it correctly:
https://github.com/dylibso/chicory/blob/0c2ed8e2e4581b83351d4796632ca18fccd17142/wasm/src/main/java/com/dylibso/chicory/wasm/WasmLimits.java#L8